xbmc/addons/kodi-addon-dev-kit/include/kodi/libKODI_inputstream.h

```diff
@@ -78,15 +74,6 @@ class CHelper_libKODI_inputstream
     libBasePath  = ((cb_array*)m_Handle)->libPath;
     libBasePath += INPUTSTREAM_HELPER_DLL;
 
-#if defined(ANDROID)
```

Comment:
apart from the fact that we use TARGET_ANDROID, the ifdef at this location is wrong